NangoHQ/nango
# feat(fleet): add node config overrides

Some customers are assigned a bigger runner instance. 

We discussed several options on Slack and I went for the pg table option

This commit adds a node_config_overrides table that allow overriding the default node config if an entry exists in this table for a given routingId.
For now there is no tooling to add/update/remove entries in this table. At first we'll update it by hand.
This mechanism can also be used to rollout a single runner for testing before rolling out all the runners, something I currently do by updating the image tag and redeploying my test runner before hitting the deploy runner button. From now it would mean adding an entry in the table.

# How to test
1. Run Nango locally with `flag:fleet:global` enabled in Redis
2. Add and/or update an entry in the `nango_runners.node_config_overrides` table
3. check that current runner is rolled out and created with correct override
